README.md

```diff
@@ -89,6 +89,16 @@ julia> rmdir(ftp, "tmp")
 julia> close(ftp)
 ```
 
+If you want to upload a file but retry on failures you can do the following:
+
+```julia
+julia> ftp_retry = retry(delays=fill(5, 4)) do
```

Comment:
In the help for `retry` they use `delays=fill(5.0, 3)` which I think reads slightly clearer.